Gemfile

```diff
@@ -4,6 +4,7 @@ source "https://rubygems.org"
 gemspec
 
 gem "mixlib-shellout", "< 3.2.3"
+gem "chef-powershell", ">= 1.0.4"
```

Comment:
if this is in the gemspec it doesn't need to be here

Reply:
it's in the GEMFILE